src/UIViewController+TransitionController.h

```diff
@@ -31,4 +32,5 @@
 @property(nonatomic, strong, readonly, nonnull) id<MDMTransitionController> mdm_transitionController
     NS_SWIFT_NAME(transitionController);
 
+@property(nonatomic, strong, nullable) id<MDMInteractiveTransitionContext> interactiveTransitionContext;
```

Comment:
We don't need to expose this if we store the variable on the transition controller.

```swift
transitionController.transition = CrossFade()
transitionController.transitionInteractionController = SwipeToDismiss()
```